### ISSUE TITLE
Align a column to the right

### ISSUE THREAD

**Issue by asliwinski** [jgthms/bulma#2440]
This is about **Bulma**.
I'm using Bulma 0.7.4

### Description

How can I right align a single column in a row without using offset? `is-pulled-right` does not work.

```
<div class="container">
  <div class="columns">
     <div class="column is-pulled-right">
      Should be pulled right
    </div>
  </div>
</div>
```

**Comment by GreiceFanka** [comment]
Hello asliwinski ! If you only have this column inside this container you can add something like this in your CSS ...

   .container{
        position:relative;
        width:30%;
        float:right;
        }
Your body is still something like this...

<div class="container">
   <div class="columns">
      <div class="column">
         <p>Should be pulled right</p>
      </div>
  </div>
</div>

and the result will look something like this...
![Capturar](https://user-images.githubusercontent.com/48022891/56033864-cf9f2f80-5d25-11e9-8e2b-9ea25d2d8290.PNG)
 
This solution can be applied if you don't want use the offset.

**Comment by ProtonScott** [comment]
@asliwinski I'm not an expert by any means, but I think what you would want to do is capture your columns in a (level)[https://bulma.io/documentation/layout/level/] 

This may require you to remove some padding via custom CSS/SASS/SCSS.

**Comment by asliwinski** [comment]
@ProtonScott I tried that, but it only works when there are both `level-left` and `level-right` present (I only need the latter). I finally solved this by pulling the div out of the `columns` div (making it a direct descendant of the `container`) and adding a custom `float: right` rule to it. Still, I believe it should be included in Bulma as a modifier (like it is in Bootstrap, for instance).

**Comment by tomhrtly** [comment]
@asliwinski 
Floats don't work with Flexbox, add "justify-content: flex-end;" to your `columns` class: https://codepen.io/tomhrtly/pen/ROXxoZ

**Comment by zitrusblau** [comment]
@tomhrtly 
I think your answer is not the solution of the question, because this would align right all columns in this row, the question was about a single column!
@asliwinski 
You can use margin-left:auto at the column element, if the column you want align right is the last column in the row, the opposite is working too, margin-right:auto if you want to align left the first column in the row.

**Comment by lukio3** [comment]
**TLDR: Add "text-align:right;" to the column.**

This is old but open... 

In the OP's case: 

```
<div class="container">
  <div class="columns">
     <div class="column" style="text-align:right;">
      Should be pulled right
    </div>
  </div>
</div>
```

This is consistent with centering content a column, where you can use text-align:center; which is exactly what the "has-text-centered" utility class does/is.

**Comment by emanuelgandrade** [comment]
that works:
```
<div class="container">
  <div class="columns">
     <div class="column" align="right">
      Should be pulled right
    </div>
  </div>
</div>
```

**Comment by theo-stocchetti** [comment]
Without the column if you use-it for align

```
<div class="container has-text-right">
  Should be pulled right
</div>
```

**Comment by mroliveira7** [comment]
As said before, the float does not work for columns but you can use it in a div inside the column:

```
<div class="columns">
  <div class="column">
    <div class="is-pulled-right">This is pulled right</div>
  </div>
</div>
```
I use it that way and works fine

**Comment by marcoczen** [comment]
2020: v0.8.2 

This worked ->  class="is-pulled-right"
This no        ->   align="right"

**Comment by pgmoir** [comment]
Old and closed, but to add clarification for those still looking. Taking inspiration from @theo-stocchetti (above) and suggestion to use `has-text-right`, but not sure why he says with container and not with column.

My requirement was to split the screen into 2 columns (3/4 and 1/4) and align buttons in the right hand one to the right. The following worked for me.

```
<div class="columns">
  <div class="column is-three-quarters">
    <!-- content here -->
  </div>
  <div class="column is-one-quarter has-text-right">
    <a class="button is-primary" href="">Add</a>
  </div>
</div>
```
